lib/puppet/module/task.rb

```diff
@@ -61,6 +62,25 @@ def self.is_tasks_filename?(path)
       return true
     end
 
+    def self.find_files(name, metadata)
+      if metadata.key?('files')
+        file_list = metadata['files'].map do |file|
+          splits = file.split(File::SEPARATOR)
```

Comment:
I think we can/should hardcode this to "/" and document that in the task spec.

Reply:
Why?

Reply:
Because with this you can't write a `files` entry that works on both Windows and Linux.

Reply:
Related, Puppet's file resource requires forward slashes: https://puppet.com/docs/puppet/5.5/types/file.html#file-attribute-path.

Reply:
The `files` key here is more like a URI as well, where forward slashes are the norm.